### ISSUE TITLE
Add Font Awesome font to AsciidoctorJ PDF

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctorj#330]
Add the following include path to the asciidoctorj-pdf/build.gradle file to include the Font Awesome fonts in the AsciidoctorJ PDF jar.

```
include "gems/prawn-icon-*/fonts/fa/*"
```

Make sure after building that the file fontawesome.ttf is present in the jar.

**Comment by abelsromero** [comment]
I think this issue could be closed since this commit fixed it https://github.com/asciidoctor/asciidoctorj/commit/143c507fe76f21755005e96227d840f5b4f96375.
Unless you want to include some test or build logic to ensure fonts are included.

Also, shouldn't be assigend to milestone 1.6.0? I mean, it is committed into that branch.

**Comment by robertpanzer** [comment]
Difficult question.
It's even already released with asciidoctorj-pdf 1.5.2-pdf-1.5.0-alpha7, but not with any "official" full release so that it would be also part of a distribution.

I am also a bit unclear about when to close issues, once the fix is released? Or when the fix is committed? 

If we release a 1.5.2.1 with the new asciidoctorj-pdf 1.5.2-pdf-1.5.0.alpha9+, then the current milestone would be ok.

**Comment by mojavelinux** [comment]
> I am also a bit unclear about when to close issues, once the fix is released? Or when the fix is committed?

I always close issues in core once the fix is committed. The milestone indicates where that fix can be found (and whether it has been released).

...in this case, we have an extra layer of complexity since we have separate version numbers for the components. That reinforces to me that we really need to think about having separate repositories. GitHub just doesn't work well when you have more than one thing to be release (much like Maven has this problem).

**Comment by mojavelinux** [comment]
...but I won't to solve the repository problem at or after the 1.6.0 release, not before since we'd need a clean starting / break point.

**Comment by robertpanzer** [comment]
OK, then I have a lot of issues to issues to close in the next days! :)

**Comment by abelsromero** [comment]
> I am also a bit unclear about when to close issues, once the fix is released? Or when the fix is committed?

It happens to me too, but I proposed because I usually see this pattern of closing on merge on GitHub.

It just happend that I went on reviewing if there was something I could fix, and saw many issues already fixed. Imho, keeping issues open untill release does not offer a real image of the project, especially with all the work done by @robertpanzer.
